pilot/pkg/model/authorization.go

```diff
@@ -135,19 +140,30 @@ func (policy *AuthorizationPolicies) ListServiceRoleBindings(ns string) map[stri
 	return rolesAndBindings.Bindings
 }
 
-// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in the given namespace.
-func (policy *AuthorizationPolicies) ListAuthorizationPolicies(ns string, workloadLabels labels.Collection) []Config {
+// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in root namespace and the config namespace.
+func (policy *AuthorizationPolicies) ListAuthorizationPolicies(configNamespace string,
```

Comment:
Is `configNamespace` same as workload namespace?

Reply:
For now I think it's always the same, in the future I feel it could be different.

Reply:
When it could be different? If such use case is unclear, I would rename to workloadNamespace instead, to avoid misuse.